dockerfiles/diy/dockerfiles/Dockerfile.repo

```diff
@@ -0,0 +1,44 @@
+# Use the rust build image from docker as our base
+FROM rust:1.60.0 as build
```

Comment:
m1 user here; can we force platform=linux/amd64 ? :D

Reply:
I'm an M1 user and I've thought about this. When we are building we *don't* want to force platform AMD64 because we can build ARM images. I do force platform linux/amd64 when we are downloading a release from github (see Dockerfile.release) because we don't have an arm binaries there.

Reply:
gotchu, lgtm then!